source/AI.cpp

```diff
@@ -1105,13 +1213,13 @@ void AI::MoveIndependent(Ship &ship, Command &command) const
 		vector<int> systemWeights;
 		int totalWeight = 0;
 		const set<const System *> &links = ship.Attributes().Get("jump drive")
-			? ship.GetSystem()->Neighbors() : ship.GetSystem()->Links();
+			? origin->Neighbors() : origin->Links();
```

Comment:
Hmm, looks like ships with a jump drive and a hyperdrive never go through links that are too far to be neighbors.

Reply:
Neighbors() includes all Links()

Reply:
Hmm, then ships that only have a jump drive can travel farther that the jump distance when a long hyperdrive link is available.

Reply:
That is explicitly intended behavior

Reply:
Ok.